src/main/java/de/cas/mse/exercise/SumDigPower.java

```diff
@@ -5,37 +5,47 @@
 
 public class SumDigPower {
 
-	public List<Long> sumDigPow(long a, long b) {
+    List<Long> eurekaList = new ArrayList<Long>();
 
-		List<Long> result = new ArrayList<Long>();
+	public List<Long> sumDigPow(long lowerLimit, long upperLimit) {
 
-		for (long i = a; i < b; i++) {
+		for (long currentNumber = lowerLimit; currentNumber < upperLimit; currentNumber++) {
 
-			List<Long> longs = new ArrayList<Long>();
+			List<Long> currentNumberSplitted = new ArrayList<Long>();
```

Comment:
`currentNumberSplitted` should be moved inside of splitValues because its hard to see the side effects